src/Symfony/Component/DependencyInjection/Tests/Compiler/IntegrationTest.php

```diff
@@ -394,10 +394,6 @@ public function testTaggedServiceWithIndexAttributeAndDefaultMethod()
 
     public function testLocatorConfiguredViaAttribute()
     {
-        if (!property_exists(SubscribedService::class, 'type')) {
-            $this->markTestSkipped('Requires symfony/service-contracts >= 3.2');
```

Comment:
`symfony/service-contracts: ^3.5` is required.

https://github.com/symfony/symfony/blob/bff8e0889497f5e722681efda730d1f078848427/src/Symfony/Component/DependencyInjection/composer.json#L18-L22